app/assets/main/react_components/footprint_form/OptionNumerical.jsx

```diff
@@ -4,18 +4,26 @@ import AnswerButton from './AnswerButton.jsx';
 const OptionNumerical = ({ onAnswerGiven, isCarOption }) => {
   const [value, setValue] = useState("");
 
+  const handleKeyDown = (event) => {
+    if (event.key === 'Enter') {
+      setValue("");
+      /* .replace() makes sure that trailing commas (, or .) are removed */
+      onAnswerGiven(value.replace(/[,.]\s*$/, ""));
+    }
+  }
+
   return (
     <div className="flex flex-col m-lg:flex-row" >
       <label className="input mb-3 m-lg:mb-0 m-lg:mr-3 flex">
-        <input autoFocus type="text" min="0" pattern="[0-9]+[.,]?[0-9]*" max="2147483647" size="7" className="flex-1" value={value} 
+        <input autoFocus type="text" min="0" pattern="[0-9]+[.,]?[0-9]*" max="2147483647" size="7" className="flex-1" value={value} onKeyPress={handleKeyDown} 
```

Comment:
Tänker att vi ska vara konsekventa med att antingen deklarera funktioner vid själva taggen, eller utanför (som du har nu). Så du kanske kan välja något av alternativen! Ger ena alternativet här.

```suggestion
        <input autoFocus type="text" min="0" pattern="[0-9]+[.,]?[0-9]*" max="2147483647" size="7" className="flex-1" value={value} 
              onKeyPress={(e) => {
                if (event.key === 'Enter') {
                    setValue("");
                    /* .replace() makes sure that trailing commas (, or .) are removed */
                    onAnswerGiven(value.replace(/[,.]\s*$/, ""));
                }
              }} 
```

Reply:
Fixed!

Reply:
Jag har googlat nu, och hittar ingenting om best practice. Jag har känslan av att man definierar korta funktioner inuti och om de är långa utanför, den här är någonstans i mitten så man kan nog göra som man vill. Men jag har definierat mest utanför så just nu är vi inte konsekventa ändå!